src/DynamoApplications/StartupUtils.cs

```diff
@@ -190,9 +200,48 @@ private static IUpdateManager InitializeUpdateManager()
             return um;
         }
 
-        
+        /// <summary>
+        /// Use this overload to construct a DynamoModel when the location of ASM to use is known and host name is known.
+        /// </summary>
+        /// <param name="CLImode">CLI mode starts the model in test mode and uses a seperate path resolver.</param>
+        /// <param name="asmPath">Path to directory containing geometry library binaries</param>
+        /// <param name="hostName">Dynamo variation identified by host.</param>
+        /// <returns></returns>
+        public static DynamoModel MakeModel(bool CLImode, string asmPath, string hostName)
```

Comment:
should the other makeModel methods be obsoleted?

Reply:
@mjkkirschner Yup, I can obsolete them